flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/translators/SinkTransformationTranslator.java

```diff
@@ -7,7 +7,7 @@
  * "License"); you may not use this file except in compliance
  * with the License.  You may obtain a copy of the License at
  *
- * http://www.apache.org/licenses/LICENSE-2.0
+ *     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
This was changed automatically by my IDE because I touched the file. In general, it seems our license includes this indentation [1], and probably the next one touching this file runs into the same issue. Do you think I should make a separate commit for this change? 

[1] https://nightlies.apache.org/flink/flink-docs-master/docs/flinkdev/ide_setup/#copyright-profile

Reply:
Very thanks for the explanation~ It seems we have many classes with the inconsistency copyright, thus perhaps we fix it in a separate PR ?

Reply:
Unfortunately every time I save it in the IDE (Intellij) it is automatically updated 🙈